File: JavaScript/02charCounter.js
//2. - count character repetitions
var text = "sdgfopgodfkgmsdkofnishndfoiphnfodjghnfjha"
var char = 'f'

var sum = 0

for (var i = 0; i < text.length; i++){
    if (text[i].toLowerCase() == char){
        sum++;
    }
}
console.log(char,'character appears',sum,'times')

File: Python/02charCounter.py
#2. - count character repetitions
text = "sdgfopgodfkgmsdkofnishndfoiphnfodjghnfjha"
char = 'f'

sum = 0

for i in text:
    if i == char:
        sum+=1
print(char,'character appears',sum,'times')